api-reference/_hidden/PdfCell/font/font.md

```diff
@@ -4,7 +4,215 @@ type: Object
 ---
 ---
 ##### shortDescription
+Object containing information about the **size**, **name** and **style** of the used font.
```

Comment:
Object that contains information about the **size**, **name** and **style** of the used font.